lib/elixir/lib/macro.ex

```diff
@@ -2642,6 +2642,15 @@ defmodule Macro do
     end
   end
 
+  defp dbg_ast_to_debuggable({:if, _meta, [condition_ast, _clauses]} = ast) do
```

Comment:
I think we might need to check the meta here - since `if` is not a special form unlike `case` & co, it can be redefined.

```elixir
 import Kernel, except: [if: 2]
 import MyOtherIf
```

We'd need a test for this case too I think

Reply:
I did that but maybe there is a better way to implement it?